server/src/main/java/org/candlepin/service/impl/DefaultContentAccessCertServiceAdapter.java

```diff
@@ -223,6 +233,41 @@ public X509Certificate createX509Certificate(Consumer consumer, BigInteger seria
         return x509Cert;
     }
 
+    /**
+     * Checks if the specified consumer is capable of using v3 certificates
+     *
+     * @param consumer
+     *  The consumer to check
+     *
+     * @return
+     *  true if the consumer is capable of using v3 certificates; false otherwise
+     */
+    private boolean consumerIsCertV3Capable(Consumer consumer) {
```

Comment:
This method is duplicated in AutobindRules.java. Let's move this somewhere common.

Reply:
You're not wrong, but at the time of writing, we don't have a consumer controller/manager and this type of logic doesn't belong in either curator. We could roll a new controller here, but it'd be for a method that's used in two places -- not sure it's worth the refactor quite yet.

Reply:
Fair enough. Let's leave it for now.